rllib/core/rl_module/torch/torch_rl_module.py

```diff
@@ -156,6 +132,20 @@ def _inference_only_get_state_hook(
         """
         pass
 
+    def _get_default_action_dist_class(self, what: str) -> Type[TorchDistribution]:
+        # The default implementation is to return TorchCategorical for Discrete action
+        # spaces and TorchDiagGaussian for Box action spaces. For all other spaces,
+        # raise a NotImplementedError
+        if isinstance(self.config.action_space, gym.spaces.Discrete):
```

Comment:
Why not using  `TorchMultiCategorical` and `TorchMultiDistribution` - things that get assembled inside of the `Catalog`?

Reply:
Let's continue brainstorming how to simplify the general RLModule experience for the user ...